.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "octref.vetur",
```

Comment:
It's recommended by vue.js https://v3.vuejs.org/guide/migration/introduction.html#ide-support
For vscode users, it will be easier to have that file to have recommendation to install the extension.